nephio-controllers/app/controller/clusterrole-bootstrap.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  annotations: {}
+  creationTimestamp: null
+  labels:
+    app.kubernetes.io/component: tbd
```

Comment:
at some point we should clean up all these labels and things that we're not using. but it's not critical or blocking for this PR (or R1 for that matter)